charts/single-node-values.yml

```diff
@@ -0,0 +1,24 @@
+ingress:
+  enabled: true
+  hosts:
+    - ""
+cloudserver-front:
+  orbit:
+    enabled: true
+
+zenko-zookeeper:
+  servers: 1
+  resources:
+    requests:
+      cpu: 500m
+      memory: 256Mi
+  heap: "256M"
+
+prometheus:
+  rbac:
+    create: false
```

Comment:
Purely because of this, I don't really like this file to be called `single-node-values.yaml`, because indeed it would fail to deploy on a single-node MetalK8s cluster, or most other single-node deployments of a recent K8s version, including GKE.

Could, instead

- *or* Minikube be deployed with RBAC enabled (`--extra-config=apiserver.Authorization.Mode=RBAC`), and things still work,
- *or* this file be renamed to something like `minikube-values.yaml`
- *or* better suggestions :smile:

Reply:
Yeah I'll play with the RBAC on minikube this way we don't potentially need another values.yml

Reply:
So I tested with RBAC and seems to work just fine but it add more steps plus makes the current steps seem a bit more cumbersome.

So this would be the modified steps plus the new ones.
```
minikube start --kubernetes-version=v1.9.0 --memory 4096 --extra config=apiserver.Authorization.Mode=RBAC

kubectl create clusterrolebinding add-on-cluster-admin --clusterrole=cluster-admin --serviceaccount=kube-system:tiller

helm init --service-account tiller --wait
```
Which I'm okay with doing but might slightly take away from the "simplicity" perceptions. 

Another option would be to append the instructions in a new section "Installing with RBAC enabled". This way we can take out the prometheus.rbac.false from the yaml file (making it more generically applicable) and simply add it as a --set in the helm install options.